### PR TITLE
Cover release/7.0 in our pipelines

### PR DESCRIPTION
**Pull request by dougbu**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -62,4 +62,15 @@ updates:
     labels:
```
Comment (dougbu) [inline .github/dependabot.yml:62]: This file doesn't matter except in 'main' branch but I wanted to keep these changes together

**Comment by dougbu** [comment]
Given https://github.com/dotnet/aspnetcore/blob/main/.github/fabricbot.json#L2773 doesn't seem to do anything and the servicing project isn't important for release/7.0 yet, I don't think fabricbot.json needs changes yet. Thoughts @dotnet/aspnet-build and @dotnet/aspnet-admins❔

File: .azure/pipelines/quarantined-pr.yml [hunk 2]
```diff
@@ -20,6 +21,7 @@ pr:
     - main
     - release/5.0
```
Comment (wtgodbe) [inline .azure/pipelines/quarantined-pr.yml:22]: Can we remove 5.0 now?

File: .azure/pipelines/helix-matrix.yml [hunk 3]
```diff
@@ -13,6 +13,7 @@ schedules:
     include:
     - release/5.0
```
Comment (wtgodbe) [inline .azure/pipelines/helix-matrix.yml:14]: Same here

**Review by wtgodbe — APPROVED** [review]

**Comment by wtgodbe** [comment]
> I don't think fabricbot.json needs changes yet.

I think that's fine

File: .azure/pipelines/quarantined-pr.yml (hunk 2 above)
Comment (dougbu) [reply .azure/pipelines/quarantined-pr.yml]: done

File: .azure/pipelines/helix-matrix.yml (hunk 3 above)
Comment (dougbu) [reply .azure/pipelines/helix-matrix.yml]: done

**Review by wtgodbe — APPROVED** [review]

**Pull request merged**